python/GangaAtlas/scripts/athena

```diff
@@ -380,10 +381,28 @@ if opt.input_dataset or opt.tag_input_dataset or opt.tag_input_files or opt.inpu
                            logger.error('File %s does not exist',pfn_list)
                            sys.exit(1)
 
-                        f = open(pfn_list)
+                        pfn_file = open(pfn_list)
+
+                        # check for nth_field option
                         pfn_list = os.path.basename(pfn_list)
-                        j.splitter.output_loc_to_input[pfn_list] = []
-                        for ln in f.readlines():
+                        if opt.nth_field_as_outdir:
+
+                            # grab the field numbers
+                            field_nums = [ int(f.strip()) for f in opt.nth_field_as_outdir.split(',') ]
```

Comment:
Apologies - saw the thumbs up and merged! I'll add this change to another Atlas PR...